apps/microtvm/gemmini/template_project/crt_config/crt_config.h

```diff
@@ -0,0 +1,57 @@
+/*
```

Comment:
We no longer check in the crt_config.h per project, instead you can use https://github.com/apache/tvm/blob/main/cmake/utils/CRTConfig.cmake to generate crt_config for this project at build time.
Here is the PR that made this change: https://github.com/apache/tvm/pull/13955

Reply:
Thanks, I changed it to use the generation at build time.